src/cephadm/cephadm

```diff
@@ -2755,6 +2757,10 @@ def _get_parser():
         '--allow-overwrite',
         action='store_true',
         help='allow overwrite of existing --output-* config/keyring/ssh files')
+    parser_bootstrap.add_argument(
+        '--allow-fqdn-hostname',
```

Comment:
imo `$ hostname` should **never** return an fqdn. What bout removing this flag, until someone demands it? cc @capri1989

Reply:
fwiw if we remove this ability, then we should also adjust the 'orchestrator host add' check, which currently allows a fqdn (as long as it matches the configured hostname).